web-client/.gitignore

```diff
@@ -1,3 +1,4 @@
 # local env files
 .env.local
 .env.*.local
+node_modules
```

Comment:
A node_modules folder kept getting created, I think when running the web-client's tests. I don't if I mess up with something

Reply:
yeah, I got it too, it's for a cache of some build files (as if we don't have temporary dir dedicated to that), it's not triggering anymore with the bump of vue, let's add it for now